synchronization-service/src/main.py

```diff
@@ -31,15 +32,10 @@ async def send_unsychronized_votes(votes) -> requests.Response:
     my_private_key = requests.get('http://web/temporary_key_location/private_key.txt').text
 
     for vote in votes:
-        new_vote = {
-            'token' : vote['vote']['token'],
-            'election_id' : vote['vote']['election_id'],
-            'party_id' : int(vote['vote']['party_id']),
-            'candidates_ids': vote['vote']['candidates'],
-        }
+        new_vote = Vote(**vote['vote'])
```

Comment:
Mohli by sme vote z parametra premenovat, nech to tu nemame vnorene: vote['vote']. Dala by som nieco neutralne ako `data`.